Optimization of the website

blvckspider opened the pull request.

blvckspider committed:
optimization of all site

blvckspider committed:
Added json file

added a json file for the displaying of seats

blvckspider commented:
Main changes: 
- Added a json file for the displaying of the isles' seats.

Changes to do yet:
- Improve seats' menu on mobile.

clibroia commented:
Thanks for your submit. Can you please remove the console.logs in your code and send a pull request that only updates the app with the JSON file, without modifying other parts of the code?

blvckspider committed:
Added only json file

Added only json file, without any changes to the original code.

The json file is still to be integrated in the app, if you want to integrate it you can find my code in the previous commit or let me know if you want a more optimized version to implement it on the original code.
